spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -1179,7 +1179,12 @@ def linenumberarea_paint_event(self, event):
         active_line_number = active_block.blockNumber() + 1
 
         def draw_pixmap(ytop, pixmap):
-            painter.drawPixmap(0, ytop + (font_height-pixmap.height()) / 2,
+            if is_pyqt46:

```

Comment:
I think we need 

```python
from qtpy import PYQT4, PYQT5

 if PYQT4:
```


https://github.com/spyder-ide/qtpy/blob/master/qtpy/__init__.py#L94